index.bs

```diff
@@ -318,6 +318,12 @@ but in some cases the feature should also be detectable
 in the language where it is used
 (such as ''@supports'' in CSS).
 
+Note that some features should not be detectable.
+For instance,
+sites should not be able to detect
+if [private browsing mode](https://www.w3.org/2001/tag/doc/private-browsing-modes/) is engaged,
+or if assistive technologies are being used. See [[#do-not-expose-use-of-private-browsing-mode]], [[#do-not-expose-use-of-assistive-tech]], and the [Observations on Private Browsing Modes](https://www.w3.org/2001/tag/doc/private-browsing-modes/) TAG Finding.
```

Comment:
I think it might be worth trying to draw a distinction here about what these features are.  In particular, I think the things we're saying should be detectable are things that add capabilities that sites can use, whereas the things we're saying shouldn't be detectable add features for end users of the web rather than for developers.

Reply:
I've more or less copied you verbatim in 2e715a6, @dbaron.